controller/SynchronisationApi.php

```diff
@@ -206,10 +207,28 @@ public function confirmSyncFinished()
             $eventManager->trigger(new SyncFinishedEvent($syncParams, $report));
 
             $this->returnJson(['message' => 'Confirmation received.']);
-        } catch (\Exception $e) {
-            $eventManager->trigger(new SyncFinishedEvent($syncParams, $report));
-            $this->returnFailure($e);
-        }
+        } catch (\Exception $e) {}
+    }
+
+    /**
+     * Receive confirmation message from client that synchronization failed on VM.
+     */
+    public function confirmSyncFailed()
+    {
+        $syncParams = [];
+        $report = \common_report_Report::createInfo('Synchronization failed.');
+        $eventManager = $this->getServiceLocator()->get(EventManager::SERVICE_ID);
+        try {
+            $this->assertHttpMethod(\Request::HTTP_POST);
+            $parameters = $this->getInputParameters();
+
+            if (isset($parameters[self::PARAM_PARAMETERS])) {
+                $syncParams = $parameters[self::PARAM_PARAMETERS];
+            }
+            $eventManager->trigger(new SyncFailedEvent($syncParams, $report));
+
+            $this->returnJson(['message' => 'Confirmation received.']);
+        } catch (\Exception $e) {}
```

Comment:
I don't like that all exceptions are muted now

Reply:
```
            $this->logError($e->getMessage());
```
?

Reply:
Fixed